test/passing/tests/doc_comments-after.ml.ref

```diff
@@ -266,11 +266,9 @@ module A = struct
   end
 end
 
-(* Same with get_pure, except that when we have both "x = t" and "y = t"
-   where t is a primed ident, * we add "x = y" to the result. This is crucial
-   for the normalizer, as it tend to drop "x = t" before * processing "y =
-   t". If we don't explicitly preserve "x = y", the normalizer cannot pick it
-   up *)
+(* Same with get_pure, except that when we have both "x = t" and "y = t" where t is a primed ident,
+ * we add "x = y" to the result. This is crucial for the normalizer, as it tend to drop "x = t" before
+ * processing "y = t". If we don't explicitly preserve "x = y", the normalizer cannot pick it up *)
```

Comment:
It seems that this kind of asterisk prefixed comments were never handled correctly before.

Reply:
Nice. I don't know what was going on before, but it is good to no longer wrap such banner comments and leave stray `*` characters in the middle of the comment.